gateway/gateway.go

```diff
@@ -422,7 +446,7 @@ func (gateway *HandleT) userWebRequestWorkerProcess(userWebRequestWorker *userWe
 			//Should be function of body
 			newJob := jobsdb.JobT{
 				UUID:         id,
-				UserID:       gjson.GetBytes(body, "batch.0.anonymousId").Str,
+				UserID:       gjson.GetBytes(body, "batch.0.userId").Str,
```

Comment:
Let's keep anonymousID as the UserID? That's the assumption so far. It's safe to keep it that way, unless needed to change.